src/reliability.jl

```diff
@@ -0,0 +1,78 @@
+using LinearAlgebra: Diagonal, isposdef
+
+struct Reliability{T <: Real}
+    alpha::T
+    dropped::Vector{Pair{Int, T}}
+end
+
+function Base.show(io::IO, x::Reliability)
```

Comment:
maybe add a test of `show` (you can check in DataFrames.jl repo how testing of display can be done)?